draft-ietf-dnsop-svcb-https.md

```diff
@@ -1702,6 +1702,178 @@ the authoritative.  Thus, splitting the RR type would double, or in
+    \x03foo\x07example\x03com\x00                      # target
+    \x00\x03                                           # key 3
+    \x00\x02                                           # length: 2 bytes
+    \x00\x35                                           # value
+
+This example has a key that is not registered, its value is unquoted.
+
+    1 foo.example.com key667=hello
+
+    \# 28 (
+    00 01                                              ; priority
+    03 66 6f 6f 07 65 78 61 6d 70 6c 65 03 63 6f 6d 00 ; target
+    02 9b                                              ; key 667
+    00 05                                              ; length 5
+    68 65 6c 6c 6f                                     ; value
+    )
+
+    \x00\x01                                           # priority
+    \x03foo\x07example\x03com\x00                      # target
+    \x02\x9b                                           # key 667
+    \x00\x05                                           # length 5
+    hello                                              # value
+
+This example has a key that is not registered, its value is quoted and contains
+a decimal-escaped character.
+
+    1 foo.example.com key667="hello\210qoo"
+
+    \# 32 (
+    00 01                                              ; priority
+    03 66 6f 6f 07 65 78 61 6d 70 6c 65 03 63 6f 6d 00 ; target
+    02 9b                                              ; key 667
+    00 09                                              ; length 9
+    68 65 6c 6c 6f d2 71 6f 6f                         ; value
+    )
+
+    \x00\x01                                           # priority
+    \x03foo\x07example\x03com\x00                      # target
+    \x02\x9b                                           # key 667
+    \x00\x09                                           # length 9
+    hello\xd2qoo                                       # value
+
+Here, two IPv6 hints are quoted in the presentation format.
+
+    1 foo.example.com ipv6hint="2001:db8::1,2001:db8::53:1"
+
+    \# 55 (
+    00 01                                              ; priority
+    03 66 6f 6f 07 65 78 61 6d 70 6c 65 03 63 6f 6d 00 ; target
+    00 06                                              ; key 6
+    00 20                                              ; length 32
+    20 01 0d b8 00 00 00 00 00 00 00 00 00 00 00 01    ; first address
+    20 01 0d b8 00 00 00 00 00 00 00 00 00 53 00 01    ; second address
+    )
+
+    \x00\x01                                           # priority
+    \x03foo\x07example\x03com\x00                      # target
+    \x00\x06                                           # key 6
+    \x00\x20                                           # length 32
+    \x20\x01\x0d\xb8\x00\x00\x00\x00
+         \x00\x00\x00\x00\x00\x00\x00\x01              # first address
+    \x20\x01\x0d\xb8\x00\x00\x00\x00
+         \x00\x00\x00\x00\x00\x53\x00\x01              # second address
```

Comment:
```suggestion
    1 foo.example.com ipv6hint="2001:db8::1,2001:db8:ffff:ffff:ffff:ffff:203.0.113.255"

    \# 55 (
    00 01                                              ; priority
    03 66 6f 6f 07 65 78 61 6d 70 6c 65 03 63 6f 6d 00 ; target
    00 06                                              ; key 6
    00 20                                              ; length 32
    20 01 0d b8 00 00 00 00 00 00 00 00 00 00 00 01    ; first address
    20 01 0d b8 ff ff ff ff ff ff ff ff cb 00 71 ff    ; second address
    )

    \x00\x01                                           # priority
    \x03foo\x07example\x03com\x00                      # target
    \x00\x06                                           # key 6
    \x00\x20                                           # length 32
    \x20\x01\x0d\xb8\x00\x00\x00\x00
         \x00\x00\x00\x00\x00\x00\x00\x01              # first address
    \x20\x01\x0d\xb8\xff\xff\xff\xff
         \xff\xff\xff\xff\xcb\x00\x71\xff              # second address
```

A suggestion for IPv4 mapped IPv6 that also crosses the 39 characters boundry of IPv6 addresses represented as string that would be the max string size if IPv4 mapped IPv6 was not anticipated.

Reply:
Works for me (as long as that line isn't too wide for TXT format).

Reply:
Ouch...  that is unfortunately not the case as TXT format has only a width of 69 characters. The example above has 83.